src/workers/postCommentedAuthor.ts

```diff
@@ -21,22 +21,40 @@ const worker: Worker = {
         return;
       }
       const post = await comment.post;
+      if (!post?.authorId && !post?.scoutId) {
```

Comment:
Can we not generalise this somewhere?
The behaviour is twice the same in this and the commentComment?

Reply:
Good point! Let me see how I can refactor it. Thank you!